SherbertEngine/SherbertEngine.vcxproj.filters

```diff
@@ -15,354 +15,367 @@
+    <ClCompile Include="Source\XTK\Tex\DirectXTexHDR.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="SRC\GAME\Game.cpp">
+    <ClCompile Include="Source\XTK\Tex\DirectXTexImage.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="SRC\ENTITY\COMPONENT\CameraComponent.cpp">
+    <ClCompile Include="Source\XTK\Tex\DirectXTexMipmaps.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="SRC\ENTITY\COMPONENT\TextMeshComponent.cpp">
+    <ClCompile Include="Source\XTK\Tex\DirectXTexMisc.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="SRC\TTF2MESH\ttf2mesh.c">
+    <ClCompile Include="Source\XTK\Tex\DirectXTexNormalMaps.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="SRC\EDITOR\WINDOW\Console.cpp">
+    <ClCompile Include="Source\XTK\Tex\DirectXTexPMAlpha.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="SRC\ENTITY\COMPONENT\RigidBodyComponent.cpp">
+    <ClCompile Include="Source\XTK\Tex\DirectXTexResize.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="SRC\SYSTEM\PhysicsSystem.cpp">
+    <ClCompile Include="Source\XTK\Tex\DirectXTexTGA.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="SRC\SYSTEM\ModelSystem.cpp">
+    <ClCompile Include="Source\XTK\Tex\DirectXTexUtil.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="SRC\EDITOR\WINDOW\About.cpp">
+    <ClCompile Include="Source\XTK\Tex\DirectXTexWIC.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="SRC\SPLASHSCREEN\SplashScreen.cpp">
+    <ClCompile Include="Source\XTK\WICTextureLoader11.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="SRC\SYSTEM\ProjectSceneSystem.cpp">
+    <ClCompile Include="Source\Editor\Window\Viewport.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="SRC\SYSTEM\ScriptingSystem.cpp">
+    <ClCompile Include="Source\ECS\Component\TransformComponent.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
-    <ClCompile Include="SRC\MODULE\Module.cpp">
+    <ClCompile Include="Source\ECS\Component\TextMeshComponent.cpp">
       <Filter>Source Files</Filter>
     </ClCompile>
   </ItemGroup>
   <ItemGroup>
-    <None Include="SRC\XTK\MATH\SimpleMath.inl">
+    <None Include="Source\XTK\Math\SimpleMath.inl">
       <Filter>Header Files</Filter>
     </None>
-    <None Include="SRC\XTK\TEX\DirectXTex.inl">
+    <None Include="Source\XTK\Tex\DirectXTex.inl">
       <Filter>Header Files</Filter>
     </None>
   </ItemGroup>
+  <ItemGroup>
+    <ResourceCompile Include="SherbertEngine.rc">
+      <Filter>Resource Files</Filter>
+    </ResourceCompile>
+  </ItemGroup>
+  <ItemGroup>
+    <Image Include="Resources\Images\SherbertEngineIcon.ico">
```

Comment:
Unsure where this came to the changes, was not addressed on my Commit seems that it has been automatically added as it was not shown.